satpy/etc/readers/fci_l2_bufr.yaml

```diff
@@ -0,0 +1,1254 @@
+reader:
+  name: fci_l2_bufr
+  short_name: FCI l2 BUFR
+  long_name: MTGi Level 2 products in BUFR format
+  description: FCI L2 BUFR Product Reader
+  status: Alpha, AMV BUFR products not supported yet
+  supports_fsspec: false
+  sensors: [fci]
+  default_channels: []
+  reader: !!python/name:satpy.readers.yaml_reader.GEOFlippableFileYAMLReader
+
+file_types:
+  fci_l2_bufr_asr:
+    file_reader: !!python/name:satpy.readers.eum_l2_bufr.EumetsatL2BufrFileHandler
+    file_patterns:
+      - '{pflag}_{location_indicator},{data_designator},MTI{spacecraft_id:1d}+FCI-2-ASR-{subtype}-{coverage}-{subsetting}-{component1}-{component2}-{component3}-{purpose}-BUFR_{oflag}_{originator}_{processing_time:%Y%m%d%H%M%S}_{facility_or_tool}_{environment}_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_{processing_mode}_{special_compression}_{disposition_mode}_{repeat_cycle_in_day:>04d}_{count_in_repeat_cycle:>04d}.bin'
+  fci_l2_bufr_amv:
+    file_reader: !!python/name:satpy.readers.eum_l2_bufr.EumetsatL2BufrFileHandler
+    file_patterns:
+      - '{pflag}_{location_indicator},{data_designator},MTI{spacecraft_id:1d}+FCI-2-AMV-{subtype}-{coverage}-{subsetting}-{component1}-{component2}-{component3}-{purpose}-BUFR_{oflag}_{originator}_{processing_time:%Y%m%d%H%M%S}_{facility_or_tool}_{environment}_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_{processing_mode}_{special_compression}_{disposition_mode}_{repeat_cycle_in_day:>04d}_{count_in_repeat_cycle:>04d}.bin'
+
+datasets:
+  #----- COMMON ------------------
+  latitude:
+    name: latitude
+    key: '#1#latitude'
+    resolution: [32000, none]
+    file_type: [fci_l2_bufr_asr, fci_l2_bufr_amv]
+    standard_name: latitude
+    units: degree_north
+    fill_value: -1.e+100
+
+  longitude:
+    name: longitude
+    key: '#1#longitude'
+    resolution: [32000, none]
```

Comment:
`none` is not allowed here, breaks for example the `Scene.available_dataset_names()` helper function. I think it should be enough to simply remove it and only keep `[32000]`. Confirm with Youva that the AMV reading still works.